packages/components/molecules/f-rating/package.json

```diff
@@ -45,7 +45,9 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-services": "1.x"
+    "@justeat/f-globalisation": "1.x",
```

Comment:
I always forget, should these be peerDeps rather than regular dependencies?

Reply:
I think it could actually be a peer as the consuming app most likely a page component will import the globalisation anyway. Ill move it 👍